packages/metal-quartz-button/src/MetalQuartzButton.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+import MetalQuartzButtonBase from './MetalQuartzButtonBase';
+import core from 'metal';
+import Soy from 'metal-soy';
+
+import templates from './MetalQuartzButton.soy.js';
+
+/**
+ * Metal Quartz Button component.
+ */
+class MetalQuartzButton extends MetalQuartzButtonBase {
+}
+
+/**
+ * State definition.
+ * @static
+ * @type {!Object}
+ */
+MetalQuartzButton.STATE = {
```

Comment:
Hey, @jbalsas. We have a disable attribute here. https://github.com/metal/metal-quartz-components/pull/8/files/d2e9cf1d793f67c445174f03087c8441cdb795dc#diff-68f8bcc3875d81fc06327e0bcdff3b00R25

But yeah, we are missing some features like `icons` for example. I've preferred to make an initial pull request to get your thoughts before continue.